### PR TITLE
feat(ramp): updates copy on order details page for pre-transacted orders

### PR DESCRIPTION
**Pull request by georgeweiler**
## **Description**
updates copy on order details page for pre-transacted orders
## **Related issues**
n/a
## **Manual testing steps**
n/a
## **Screenshots/Recordings**
n/a
### **Before**
n/a
### **After**
n/a
## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've clearly explained what problem this PR is solving and how it is solved.
- [ ] I've linked related issues
- [ ] I've included manual testing steps
- [ ] I've included screenshots/recordings if applicable
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [ ] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [ ] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: app/components/UI/Ramp/common/components/OrderDetails.tsx [hunk 1]
```diff
@@ -155,6 +157,29 @@ const Stage: React.FC<PropsStage> = ({
...
+                  : 'fiat_on_ramp_aggregator.order_details.pending',
+              )}
+            </Text>
+            <Text small centered grey>
+              {isTransacted
+                ? pendingDescription
+                : strings(
+                    'fiat_on_ramp_aggregator.order_details.continue_order_description',
+                  )}
+            </Text>
```
Comment (wachunei) [inline app/components/UI/Ramp/common/components/OrderDetails.tsx:178]: `pendingDescription` is optional, this is rendering an empty Text here when it is not supplied.

**Review by wachunei — APPROVED** [review]
LGTM

**Comment by bkirb** [comment]
@georgeweiler LGTM I verified this change ✅

**Comment by bkirb** [comment]
@georgeweiler LGTM I verified the latest commit

**Comment by sethkfman** [comment]
LGTM

**Pull request merged**